Example/Podfile

```diff
@@ -3,6 +3,7 @@ inhibit_all_warnings!
 
 target 'SDWebImageFLPlugin_Example' do
   pod 'SDWebImageFLPlugin', :path => '../'
+  pod 'SDWebImage/Core', :path => '../../SDWebImage'
```

Comment:
we don't want to add this, it will not work. Also breaks the CI

Reply:
@bpoplauschi It's local podfile because I test it locally to ensure it works. But that https://github.com/rs/SDWebImage/pull/2453#event-1812523011 was not been merged.

Now I can specify the branch dependency and update podfile.lock

Reply:
Got it :)